src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/FixedPage.cs

```diff
@@ -615,8 +615,6 @@ internal static Uri GetLinkUri(IInputElement element, Uri inputUri)
             Debug.Assert(dpo != null, "GetLinkUri shouldn't be called for non-DependencyObjects.");
 
             if (inputUri != null)
-            // We want to allow navigation to pack:// Uris as we may get these from automation, but we 
```

Comment:
I think this comment is in regards to the logic below, not the check you removed

Reply:
Yeah, I was on the fence on whether to remove this because I think its both (it was placed right under `(inputUri.Scheme == PackUriHelper.UriSchemePack && !String.IsNullOrEmpty(inputUri.Fragment))` in order to explain why this extra check was there).

However, I guess its better to leave this in as it does describe other functionality. Re-added comment.

Reply:
this comment does seem out-of-date now, it sounds like it was referring only to the check. before the removal, the if statement read like this:

```
if (inputUri != null &&
   (CanNavigateToUri(inputUri) ||
   (inputUri.Scheme == PackUriHelper.UriSchemePack && !String.IsNullOrEmpty(inputUri.Fragment))))
```

so i think the comment was about the fact that `CanNavigateToUrl` would return false and there were scenarios where we wanted to navigate even when that returned false. but now it seems we always navigate, unless i'm misunderstanding something?

Reply:
I think you're right @stevenbrix , upon first reading it I assumed it was an overview of the logic below, but looking at it again it makes more sense, we should remove it.